assets/queries/k8s/object_is_using_a_deprecated_api_version/query.rego

```diff
@@ -21,8 +23,8 @@ CxPolicy[result] {
 		},
 	}
 
-	object.get(recommendedVersions, document.apiVersion, "undefined") != "undefined"
-	object.get(recommendedVersions[document.apiVersion], document.kind, "undefined") != "undefined"
+	common_lib.valid_key(recommendedVersions, document.apiVersion)
+	common_lib.valid_key(recommendedVersions[document.apiVersion], document.kind)
```

Comment:
It can be simplified just to `common_lib.valid_key(recommendedVersions[document.apiVersion], document.kind)`